examples/authn/static/index.html

```diff
@@ -27,5 +29,170 @@ <h1>
 			{{ .content }}
 		</div>
 	</main>
+	<script>
+		$(document).ready(function () {
+			if (!window.PublicKeyCredential) {
+				alert("Error: this browser does not support WebAuthn");
+				return;
+			}
+		});
+
+		async function checkUserExists(username) {
+			const response = await fetch('http://localhost:8000/v1beta1/auth/register/passkey?email=' + username);
+			const userData = await response.json();
+			requestType = userData.state_options.type;
+			return requestType; 
+		}
+
+		async function registerOrLoginUser() {
+			username = $("#email").val()
+			if (username === "") {
+				alert("Please enter a username");
+				return;
+			}
+			const userExists = await checkUserExists(username);
+			if (userExists == "register") {
+				registerUser();
+			} else {
+				loginUser();
+			}
+		}
+
+		async function registerUser() {
+			username = $("#email").val()
+			if (username === "") {
+				alert("please enter a username");   
+				return;
+			}
+			let flow_id = ""
+			const response = await fetch('http://localhost:8000/v1beta1/auth/register/passkey?email=' + username)
```

Comment:
let's create variables for these server path at the top of the js file just like we have for `main.go`